test/grouping.jl

```diff
@@ -3020,4 +3014,156 @@ end
     @test_throws MethodError select(gdf, AsTable([]) => ByRow(inc0) => :bin)
 end
 
+@testset "aggregation of reordered groups" begin
+    df = DataFrame(id = [1, 2, 3, 1, 3, 2], x=1:6)
```

Comment:
```suggestion
    df = DataFrame(id=[1, 2, 3, 1, 3, 2], x=1:6)
```
(For consistency with below.)